bigbluebutton-html5/imports/ui/components/polling/component.jsx

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import Button from '/imports/ui/components/button/component';
 import styles from './styles.scss';
+import { defineMessages, injectIntl } from 'react-intl';
 
-export default class PollingComponent extends React.Component {
+const intlMessages = defineMessages({
+  pollingTitleLabel: {
+    id: 'app.polling.pollingTitle',
+    description: 'Title label for polling options',
```

Comment:
This description property is not suppose to be retrieved to display or voice anything to the user. It's purpose is only to give context to the developer as to what the message id is for if they were looking through the code.